docs/datamodel/computeds.rst

```diff
@@ -33,6 +33,15 @@ field is referenced in a query.
   For more information, see the :ref:`EdgeQL > Select > Computeds
   <ref_eql_select_computeds>`.
 
+.. warning::
+
+  Volatile functions are not allowed in computed properties defined in schema.
+  This means that, for example, your schema-defined computed property cannot
+  call :eql:func:`datetime_current`, but it *can* call
+  :eql:func:`datetime_of_transaction` or :eql:func:`datetime_of_statement`.
+  This does *not* apply to computed properties in :ref:`free objects
```

Comment:
Is this exception specific to free objects or queries outside of the schema generally?

Reply:
Are there other contexts where computed properties can be used besides in schema or in a free object? Maybe I need to change it to say, "This does *not* apply to uses of computed properties outside schema."

Reply:
computeds can be used in any shape, for example
```
select User { id, name, last_login := (select .logins order by .timestamp desc limit 1) }
```
The type here is `User` but the shape includes a computed link.

Reply:
> This does not apply to uses of computed properties outside the schema.

This seems correct to me.

Reply:
> computeds can be used in any shape, for example
> 
> ```
> select User { id, name, last_login := (select .logins order by .timestamp desc limit 1) }
> ```
> 
> The type here is `User` but the shape includes a computed link.

That makes sense. Thanks for clarifying! I've updated the language to catch all usages outside of schema.